crates/router/src/consts.rs

```diff
@@ -15,6 +15,9 @@ pub const REQUEST_TIME_OUT: u64 = 30;
 ///Payment intent fulfillment default timeout (in seconds)
 pub const DEFAULT_FULFILLMENT_TIME: i64 = 15 * 60;
 
+// prefix of ephemeral_key
+pub const EPHEMERAL_KEY_PREFIX: &str = "epk";
+
```

Comment:
Replace with this const wherever "epk" is used such as while generating ephemeral key